src/yellow/components/message/_index.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../../../node_modules/@getninjas/niten-tokens/build/web/niten-tokens';
+
+.message {
+  display: flex;
+  flex-direction: row;
+  padding: $size-spacing-xs;
+
+  &__icon {
+    height: 20px;
+    margin-right: $size-spacing-xxs;
+    width: 20px;
+
+    svg {
```

Comment:
We will use the message__icon child as a `svg`?

If so we need to put this rule in the modifiers below (`--positive, --alert, --negative`) with the same color as the text and change the example to use `svg` directly in the html.

Reply:
Oops!

For now we will not use SVG no, this was just a test that I did and ended up forgetting.

In the future we will use the icons as a library.